src/main/java/ch/heigvd/pro/b04/questions/ServerQuestion.java

```diff
@@ -29,23 +29,24 @@ public ServerQuestion() {
   /**
    * Creates a new {@link ServerQuestion}.
    *
-   * @param id      The question identifier.
-   * @param index   The question index.
-   * @param title   The question title.
-   * @param details The question details.
-   * @param visible The question visibility.
-   * @param min     The lower bound for the number of required answers.
-   * @param max     The upper bound for the number of required answers.
+   * @param identifier The question identifier.
+   * @param index      The question index.
+   * @param title      The question title.
+   * @param details    The question details.
+   * @param visible    The question visibility.
+   * @param min        The lower bound for the number of required answers.
+   * @param max        The upper bound for the number of required answers.
    */
   @Builder
```

Comment:
I think we should remove this idea of inheritance.

Reply:
Me not. It is the proper way to do it and it doesn't require a lot of adaptations...